### PR TITLE
lookup: declare bankruptcy on flaky modules

### PR DESCRIPTION
**Pull request by Trott**
Remove all modules that failed the latest 20.x release run. We need to be able to trust that a failed CITGM run is something that needs to be investigated.

##### Checklist

- [x] `npm test` passes
- [x] contribution guidelines followed
      [here](https://github.com/nodejs/citgm/blob/HEAD/CONTRIBUTING.md)

### CODE REVIEW

**Review by anonrig — APPROVED** [review]

**Comment by Trott** [comment]
@GeoffreyBooth noted that 20.6.0 had problems and that I should probably use 20.5.x as the baseline for failed tests (or "failed both 20.6.0 and 20.5.x") so I'll do that.

**Comment by ljharb** [comment]
"failed" isn't the same as "flaky" tho, at least for is-core-module and resolve.

**Review by GeoffreyBooth — APPROVED** [review]

**Review by ruyadorno — APPROVED** [review]

**Comment by RafaelGSS** [comment]
Can we ping module authors here, so they are aware of this action? We should merge it anyway

**Comment by ruyadorno** [comment]
I would love for us to be able to trim down the list a bit more in order to help with build times but we can work on it in a follow-up PR.

File: lib/lookup.json [hunk 1]
```diff
@@ -424,25 +304,6 @@
     "maintainers": "TooTallNate",
     "skip": [true, "12", "win32"]
   },
-  "resolve": {
-    "prefix": "v",
-    "maintainers": "ljharb",
-    "scripts": ["tests-only", "posttest"]
-  },
```
[the remark this answers is not in the document]
Comment (Trott) [reply lib/lookup.json]: I've added `resolve` back.

File: lib/lookup.json [hunk 2]
```diff
@@ -237,29 +186,10 @@
...
   },
-  "inherits": {
-    "prefix": "v",
-    "maintainers": "isaacs"
-  },
-  "is-core-module": {
-    "prefix": "v",
-    "maintainers": "ljharb",
-    "scripts": ["tests-only"]
-  },
```
[the remark this answers is not in the document]
Comment (Trott) [reply lib/lookup.json:248]: `is-core-module` fails every CITGM run I've looked at. It's always on Windows. Maybe we can just skip Windows?

https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3205/nodes=win-vs2019/testReport/junit/(root)/citgm/is_core_module_v2_13_0/
https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3206/nodes=win-vs2019/testReport/junit/(root)/citgm/is_core_module_v2_13_0/
https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3207/nodes=win-vs2019/testReport/junit/(root)/citgm/is_core_module_v2_13_0/
https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3213/nodes=win-vs2019/testReport/junit/(root)/citgm/is_core_module_v2_13_0/

File: lib/lookup.json [hunk 3]
```diff
@@ -310,19 +233,6 @@
...
-    "maintainers": "wadey"
-  },
-  "mime": {
-    "prefix": "v",
-    "maintainers": "broofa"
-  },
-  "minimist": {
-    "npm": true,
-    "maintainers": "substack"
-  },
```
[the remark this answers is not in the document]
Comment (Trott) [reply lib/lookup.json:325]: Maybe we can just skip Windows for minimist unless and until someone gets around to looking into why it's not working on our Windows infrastructure?

https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3213/nodes=win-vs2019/testReport/junit/(root)/citgm/minimist_v1_2_8/
https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3212/nodes=win-vs2019/testReport/junit/(root)/citgm/minimist_v1_2_8/
https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3211/nodes=win-vs2019/testReport/junit/(root)/citgm/minimist_v1_2_8/

File: lib/lookup.json [hunk 4]
```diff
@@ -505,11 +353,6 @@
     "maintainers": "tj",
     "skip": ">=11"
   },
-  "tape": {
-    "head": true,
-    "prefix": "v",
-    "maintainers": "substack"
-  },
```
[the remark this answers is not in the document]
Comment (Trott) [reply lib/lookup.json:512]: Failure on RHEL:

https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3211/nodes=rhel8-ppc64le/testReport/junit/(root)/citgm/tape_v5_6_6/

Failure on macOS:

https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3212/nodes=osx1015/testReport/junit/(root)/citgm/tape_v5_6_6/

File: lib/lookup.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (ljharb) [reply lib/lookup.json:248]: That sounds like a great compromise, thank you.

File: lib/lookup.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (ljharb) [reply lib/lookup.json:325]: yes, please do!

File: lib/lookup.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (Trott) [reply lib/lookup.json:248]: Done.

**Comment by Trott** [comment]
Whoops, I pushed to main by accident and now it marked this as merged. 🙃 

Sorry. I'll force push it back to where it was in a moment and open a new PR. 😞

**Comment by nicolo-ribaudo** [comment]
This is awesome :)

**Pull request merged**